src/Plugin/TraefikEndpoints/TraefikEndpointProvider.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace my127\Workspace\Plugin\TraefikEndpoints;
+
+use my127\Workspace\File\JsonLoader;
+
+class TraefikEndpointProvider
+{
+    public const TRAEFIK_BASE_URL = 'https://my127.site';
```

Comment:
be good to get this from the env somehow than assume it exists, but for now probably "fine"